### PR TITLE
fix(lro): clarify protobuf response for no parallel requests

### PR DESCRIPTION
**Pull request by toumorokoshi**
For protobuf guidance, it was unclear response code should be.

Partially addresses #224 

## 🍱 Types of changes

What types of changes does your code introduce to AEP? _Put an `x` in the boxes
that apply_

- [ ] Enhancement
- [ ] [New proposal](https://aep.dev/1#workflow)
- [ ] Migrated from google.aip.dev
- [x] Chore / Quick Fix

## 📋 Your checklist for this pull request

Please review the [AEP Style and Guidance](https://aep.dev/style-guide) for
contributing to this repository.

### General

- [x] Basic [Guidance](https://aep.dev/style-guide#guidance) is met.
- [x] Ensure that your PR
      [references AEPs](https://aep.dev/style-guide#referencing-aeps)
      correctly.
- [x] [My code has been formatted](https://aep.dev/contributing#formatting)
      (usually `prettier -w .`)

### CODE REVIEW

File: aep/general/0151/aep.md.j2 [hunk 1]
```diff
@@ -75,10 +75,23 @@ but is not obligated to do so:
...
 - Resources that accept multiple parallel requests **may** place them in a
   queue rather than work on the requests simultaneously.
-- Resource that does not permit multiple requests in parallel (denying any new
-  request until the one that is in progress finishes) **must** return
-  `409 Conflict` if a user attempts a parallel request, and include an error
-  message explaining the situation.
+
+{% tab proto %}
+
+- A Resource that does not permit multiple requests in parallel (denying any new
```
Comment (wora) [inline aep/general/0151/aep.md.j2:81]: "multiple requests" is inaccurate. Multiple GET requests should be fine.

File: aep/general/0151/aep.md.j2 [hunk 2]
```diff
@@ -75,10 +75,23 @@ but is not obligated to do so:
...
+  request until the one that is in progress finishes) **must** return `ABORTED`
```
Comment (wora) [inline aep/general/0151/aep.md.j2:82]: "return" => "return an ABORTED error"

File: aep/general/0151/aep.md.j2 [hunk 3]
```diff
@@ -75,10 +75,23 @@ but is not obligated to do so:
...
+  if a user attempts a parallel request, and include an error message explaining
```
Comment (wora) [inline aep/general/0151/aep.md.j2:83]: We should not encourage clients to parse error messages. All errors should have an error message, there is no need to mention it here.

We can simplify the wording like this:
If a resource doesn't support concurrent mutation requests, the server must return an ABORTED error if the resource has an outstanding mutation.

File: aep/general/0151/aep.md.j2 (hunk 1 above)
Comment (toumorokoshi) [reply aep/general/0151/aep.md.j2:81]: LROs will apply to more than GET though - for example multiple updates may be requested.

File: aep/general/0151/aep.md.j2 (hunk 2 above)
Comment (toumorokoshi) [reply aep/general/0151/aep.md.j2:82]: sounds good! will address that.

File: aep/general/0151/aep.md.j2 (hunk 3 above)
Comment (toumorokoshi) [reply aep/general/0151/aep.md.j2:83]: I agree with not parsing the error. The error message guidance really should be in the *should* category anyway - since it won't block clients from consuming you.

Will update the wording based on your suggestion! thanks.

File: aep/general/0151/aep.md.j2 (hunk 1 above)
Comment (toumorokoshi) [reply aep/general/0151/aep.md.j2]: applied the suggestions! thanks.

**Comment by toumorokoshi** [comment]
@wora addressed comments! PTAL.

**Comment by dv-stewarts** [comment]
LGTM. I think splitting into tabs per protocol is the right answer. I'm not a reviewer, so cannot approve.

File: aep/general/0151/aep.md.j2 [hunk 4]
```diff
@@ -75,10 +75,18 @@ but is not obligated to do so:
...
+
+- A resource that does not support concurrent mutation requests **must** return an
+  `ABORTED` error if the resource has an outstanding mutation.
+
+{% tab oas %}
+
+- A resource that does not support concurrent mutation requests **must** respond
+  with a `409 Conflict` status code if the resource has an outstanding mutation.
+
+{% endtabs %}
```
Comment (rofrankel) [inline aep/general/0151/aep.md.j2:89]:
```suggestion
- Resource that does not permit multiple requests in parallel (denying any new
  request until the one that is in progress finishes) **must** return
  `ABORTED`/`409 Conflict` if a user attempts a parallel request, and include an error
  message explaining the situation.
```

We discussed this live - seems unfortunate to add all the tab chrome to the page just to vary a single term.  Thoughts?

Comment (toumorokoshi) [reply aep/general/0151/aep.md.j2:89]: I'm fine with consolidating it. 

@wora gave feedback on the "include an error message" guidance earlier, and suggested that adding this type of guidance may result in users using the error string as a way to figure out the actual error (concurrent request). I removed that part intentionally based on that feedback (as I also agreed). Thoughts on that?
